depends/patches/qt/drop_lrelease_dependency.patch

```diff
@@ -0,0 +1,24 @@
+commit 67b3ed7406e1d0762188dbad2c44a06824ba0778
+Author: fanquake <fanquake@gmail.com>
+Date:   Tue Aug 18 15:24:01 2020 +0800
+
+    Drop dependency on lrelease
+
+    Qts buildsystem insists on using the installed lrelease, but gets
+    confused about how to find it. Since we manually control the build
+    order, just drop the dependency.
+
+    See #9469
+
+diff --git a/qttranslations/translations/translations.pro b/qttranslations/translations/translations.pro
+index 694544c..eff339d 100644
+--- a/qttranslations/translations/translations.pro
++++ b/qttranslations/translations/translations.pro
+@@ -107,7 +107,6 @@ updateqm.input = TRANSLATIONS
+ updateqm.output = $$MODULE_BASE_OUTDIR/translations/${QMAKE_FILE_BASE}.qm
+ updateqm.commands = $$LRELEASE ${QMAKE_FILE_IN} -qm ${QMAKE_FILE_OUT}
+ silent:updateqm.commands = @echo lrelease ${QMAKE_FILE_IN} && $$updateqm.commands
+-updateqm.depends = $$LRELEASE_EXE
```

Comment:
2c5c41344976cf4aba45621a8b99e6df3a2de33b

The following change makes `busybox patch` command works correctly, and it fixes Travis ARM build:
```suggestion
-updateqm.depends = $$LRELEASE_EXE
+
```

Reply:
Hmm, it seem works for busybox patch only.